Makefile

```diff
@@ -337,6 +339,16 @@ ginkgo-subpkgs-tests: # TODO(klihub): coverage ?
 	            -r . || exit 1); \
 	done
 
+e2e-tests: build images
+	$(Q)[[ -x containerd/bin/containerd ]] || { \
+		echo "containerd/bin/containerd missing, consider:"; \
+		echo "  git clone https://github.com/containerd/containerd.git"; \
+		echo "  cd containerd && make && cd .."; \
+		exit 1; \
+	} && \
+	tests="$(if $(E2E_TESTS),$(E2E_TESTS),policies.test-suite)"; \
+        cd test/e2e && $(E2E_RUN) $$tests;
+
```

Comment:
I have an alternative fix which instead changes the ansible provisioning to fetch and install containerd from a release tarball instead of compiling it...

Reply:
I'll try to tie up the loose ends and make a PR out of it.

Reply:
https://github.com/containers/nri-plugins/pull/144

Reply:
#154 fixes this and a bunch of other e2e test bootstrapping problems. Cherry-picked and adjusted this commit on top of that.